LOAN/RealEstateLoans/ConstructionLoans.rdf

```diff
@@ -39,141 +39,140 @@
 	xmlns:sm="http://www.omg.org/techprocess/ab/SpecificationMetadata/"
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
-	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanTypes/ConstructionLoans/">
-		<rdfs:label xml:lang="en">ConstructionLoans</rdfs:label>
-		<dct:abstract>Construction loans are loans in name only, in that the concept referred to as a construction loan is effectively a credit facility, with separate draw-downs (loans as defined in these ontologies) being enabled upon evidence of completion of agreed stages of the construction project, 
-		Note that for completion this ontology will need to be extended with a number of project management concepts describing the parameters of the construction project that are referred to in the contract for this facility. Some basic project management terms such as milestones are already included but will need framing within more foundational concepts for project management.</dct:abstract>
-		<sm:fileAbbreviation>fibo-loan-typ-cl</sm:fileAbbreviation>
+	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/RealEstateLoans/ConstructionLoans/">
+		<rdfs:label xml:lang="en">Construction Loans Ontology</rdfs:label>
+		<dct:abstract>Construction loans are loans in name only, in that the concept referred to as a construction loan is effectively a credit facility, with separate draw-downs (loans as defined in these ontologies) being enabled upon evidence of completion of agreed stages of the construction project. Note that for completion this ontology will need to be extended with a number of project management concepts describing the parameters of the construction project that are referred to in the contract for this facility. Some basic project management terms such as milestones are already included but will need framing within more foundational concepts for project management.</dct:abstract>
```

Comment:
No I don't think we should attempt an ontology for project management!

Reply:
@rivettp - Agreed - let's address construction loans with an SME in another issue.